_rules/SC1-3-1-heading-name-does-not-only-contain-spaces.md

```diff
@@ -0,0 +1,155 @@
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to any HTML element with the [semantic role](#semantic-role) of `heading` that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+None of the target elements has an [accessible name](#accessible-name) that contains only spaces.
+
+**Note:** In the [Accessible Name and Description Computation](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te) all carriage returns, newlines, tabs, and form-feeds are replaced with a single space.
+
+## Assumptions
+
+This rule assumes that having an element that unintentionally shows up programatically to a user of assistive technologies as a heading element, but is not shown visually as a heading on the page, is a violation of WCAG success criterion 1.3.1 Info and Relationships.
+
+## Accessibility Support
+
+Handling of headings containing only spaces, space characters, carriage returns, newlines, tabs, and form-feeds varies between different assistive technologies and browsers. This means that even though the outcome of this rule is *failed*, users of certain assistive technology and browser combinations might not experience an issue.
+
+## Background
+
+- In some screen reader and browser combinations, headings containing only spaces, space characters, carriage returns, newlines, tabs, and form-feeds will show up as empty headings, confusing the user experience. For a screen reader user it will be hard to get an overview of the heading structure of the page, if "unused" headings are included in the heading structure, and it can be hard to know if the heading is just white space that has accidentally been marked up as a heading, or if it is an actual heading that for some reason doesn't have an accessible name. Since this is a case where the programatically determinable structure of the page doesn't match the visual presentation, this is a violation of success criterion [1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships).
+- [Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [The br element](https://www.w3.org/TR/html/textlevel-semantics.html#the-br-element)
+- [Unicode Characters in the 'Separator, Space' Category](https://www.fileformat.info/info/unicode/category/Zs/list.htm)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+`h2` element is empty.
+
+```html
+<h2></h2>
+```
+
+#### Passed example 2
+
+Element with the semantic role of heading is empty.
+
+```html
+<div role="heading"></div>
+```
+
+#### Passed example 3
+
+`h2` element has other content than spaces.
+
+```html
+<h2>'</h2>
+```
+
+### Failed
+
+#### Failed example 1
+
+`h2` element contains `br` element as only content.
+
+```html
+<h2><br /></h2>
+```
+
+#### Failed example 2
+
+`h2` contains `&nbsp;`(no break space character) as only content.
+
+```html
+<h2>&nbsp;</h2>
```

Comment:
I don't think this would fail, going by what is and isn't replaced with "space" characters. May need to clarify exactly which chars should fail.